crates/uv-workspace/src/workspace.rs

```diff
@@ -674,6 +674,21 @@ impl ProjectWorkspace {
         &self.workspace().packages[&self.project_name]
     }
 
+    /// Returns the workspace root member as a [`WorkspaceMember`].
+    pub fn root_member(&self) -> &WorkspaceMember {
+        self.workspace()
+            .packages
+            .iter()
+            .find_map(|(_, member)| {
+                if member.root() == self.workspace().install_path() {
+                    Some(member)
+                } else {
+                    None
+                }
+            })
+            .expect("workspace root member must exist")
```

Comment:
is this true for a virtual workspaces?

Reply:
It is, every workspace has a root member, either explicit or implicit.